### DISCUSSION TITLE
Not able to Add project in Helix Sidekick in MacOS 12.2 m1

### DISCUSSION THREAD

**Discussion by nikhilkumar-24** [adobe/helix-home#297] [Q&A] [answered]
Hi Team,

Is was working on the Beta version of Project Helix. Facing issue while adding the project in Helix Sidekick using Chrome plugin. On click of "Add this Project" in the chrome plugin nothing happens.

Getting below error in console. (Screenshot 1)

Also tried configuring the project using the url (https://www.hlx.live/tools/sidekick/) but it seems to be throwing console on click of Add Project too. (Screenshot 2)
![4B7E1674-BB7B-43EC-84EC-C3CA8396E0F3](https://user-images.githubusercontent.com/115579540/195172867-14f57979-db6e-492c-8289-03f4349d5922.jpeg)
![DAD680D9-32A4-4C2E-A09F-0A2E7C37DD90](https://user-images.githubusercontent.com/115579540/195172877-f87ef18d-007e-4e99-9f22-d431c0724717.jpeg)

**Comment by kptdobe** [comment]
@nikhilkumar-24 I can add your project to my sidekick:

![image](https://user-images.githubusercontent.com/474200/195269406-7332ac48-aa06-498d-9e66-78939e28246a.png)

Same if I use https://www.hlx.live/tools/sidekick/?giturl=https%3A%2F%2Fgithub.com%2Fnikhilkumar-24%2Fmy-website&project=Alex+Test

The first stack trace you've shared is GitHub related, I also see it on page load too, completely unrelated to the sidekick or to Helix.
I do not see the second stack trace.

When you tested, wasn't your GitHub project private ? and now it is public... ?

**Reply by nikhilkumar-24** [reply]
@kptdobe - It was always public.

**Comment by kptdobe** [comment]
Also, what is the version of your sidekick ? `Sidekick > Manage Extension` you'll see the version here.

**Reply by nikhilkumar-24** [reply]
@kptdobe  - I am using 5.9.0.

**Comment by kptdobe** [comment]
humm... this is weird. Could you open the `background.html` inspector ? `Sidekick > Manage Extension` and just click on the `background.html` link. You should see the console for the extension itself. Then go back to your Chrome tab on https://github.com/nikhilkumar-24/my-website and click on "Add Project" again. Any logs / error message ?

**Reply by nikhilkumar-24** [reply]
@kptdobe - In console of background.html it shows same set of errors.
![8F8E5D6E-2193-4D93-B4D0-8528942C3B10](https://user-images.githubusercontent.com/115579540/195294915-9de113a9-eab5-446c-aa45-61b84d7d9349.jpeg)
![177029FE-1C9B-4525-B047-CA3846A636DA](https://user-images.githubusercontent.com/115579540/195294942-72f4f92b-8e0c-4802-ad09-bcab4754a8f7.jpeg)

**Comment by kptdobe** [comment, marked as answer]
What does `curl -v https://raw.githubusercontent.com/nikhilkumar-24/my-website/main/tools/sidekick/config.js` do from your computer ?
Actually, it seems you cannot connect to https://raw.githubusercontent.com/ (DNS error). I get a 404 error for this url, which is expected but not a fetch error.
If you can curl the host, it should be something wrong in your browser. If not, this is something with your network.

**Reply by nikhilkumar-24** [reply]
@kptdobe - Okay, let me have a look on another chrome browser on different machine. Is it due to any permission access on the browser?

**Reply by nikhilkumar-24** [reply]
@kptdobe - Okay, let me have a look on another chrome browser on different machine. Is it due to any permission access on the browser?

**Reply by nikhilkumar-24** [reply]
@kptdobe - Okay, let me have a look on another chrome browser on different machine. Is it due to any permission access on the browser?

**Reply by nikhilkumar-24** [reply]
@kptdobe  - It's resolved now, I connected to my Company's VPN and it seems to working fine now.
Thanks!
